k8s/overlays/coreso/prod/postgresql-configmap.yaml

```diff
@@ -14,12 +14,18 @@ data:
     CREATE USER $POSTGRES_CONFIG_USER WITH ENCRYPTED PASSWORD '$POSTGRES_CONFIG_PASSWORD';
     GRANT ALL PRIVILEGES ON DATABASE config_server_db TO $POSTGRES_CONFIG_USER;
 
+    CREATE DATABASE cse_export_idcc_tasks;
+    CREATE USER $POSTGRES_CSE_EXPORT_IDCC_USER WITH ENCRYPTED PASSWORD '$POSTGRES_CSE_EXPORT_IDCC_PASSWORD';
+    GRANT ALL PRIVILEGES ON DATABASE cse_export_idcc_tasks TO $POSTGRES_CSE_EXPORT_IDCC_USER;
     CREATE DATABASE cse_import_idcc_tasks;
     CREATE USER $POSTGRES_CSE_IMPORT_IDCC_USER WITH ENCRYPTED PASSWORD '$POSTGRES_CSE_IMPORT_IDCC_PASSWORD';
     GRANT ALL PRIVILEGES ON DATABASE cse_import_idcc_tasks TO $POSTGRES_CSE_IMPORT_IDCC_USER;
     CREATE DATABASE cse_import_d2cc_tasks;
     CREATE USER $POSTGRES_CSE_IMPORT_D2CC_USER WITH ENCRYPTED PASSWORD '$POSTGRES_CSE_IMPORT_D2CC_PASSWORD';
     GRANT ALL PRIVILEGES ON DATABASE cse_import_d2cc_tasks TO $POSTGRES_CSE_IMPORT_D2CC_USER;
+    CREATE DATABASE cse_import_ec_idcc_tasks;
```

Comment:
Are you sure that we don't add cse_import_ec_d2cc also?

Reply:
yes